### ISSUE TITLE
Customize 'stroke-width' in column chart.

### ISSUE THREAD

**Issue by nicolabortignon** [n3-charts/line-chart#309]
Would be great if we can set the stroke-width in the options. 

At the moment is automatically set in this way:  'stroke-width': '1px'.
I'm looking to removing it completely, so technically a 0px, will do the job here.

**Comment by chaosmail** [comment]
Hi Nicola,
We have the `thikness` option (not the best name) to enable this for line and area charts. Maybe this could also work for column charts.
Best,
Christoph

**Comment by nicolabortignon** [comment]
That sounds good to me!

**Comment by chaosmail** [comment]
Ni Nicola,
let me know how it is going with this!
Best,
Christoph

**Comment by adonespitogo** [comment]
Is this available in V2?

**Comment by lorem--ipsum** [comment]
Yes, through CSS. Take a look at `https://github.com/n3-charts/line-chart/blob/master/src/styles/_lines.scss` !
